nuxeo/templates/deployment.yaml

```diff
@@ -52,7 +57,7 @@ spec:
             cpu: "{{ .Values.resources.limits.cpu }}"
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
 {{- if .Values.image.pullPolicy }}
-        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        imagePullPolicy: {{ ternary ("Always") (.Values.image.pullPolicy) (or (eq "latest" (lower .Values.image.tag)) (hasSuffix "snapshot" (lower .Values.image.tag))) }}
```

Comment:
if the docker images version is a mutable(`latest` or `-SNAPSHOT` version), imagepull policy is `Always` as the image is updated with the same version in dev cycle